spec/index.bs

```diff
@@ -192,34 +192,81 @@ dictionary GPULimits {
 </script>
 
 
-Buffers {#buffers}
-==================
+# {{GPUBuffer}} # {#GPUBuffer}
 
-## GPUBuffer ## {#buffer}
+A {{GPUBuffer}} represents a linear allocation of memory to hold data that can be used by GPU operations.
+Each byte of the allocation can be addressed by its offset from the start of the {{GPUBuffer}}.
 
-<script type=idl>
-interface GPUBuffer : GPUObjectBase {
-    Promise<ArrayBuffer> mapReadAsync();
-    Promise<ArrayBuffer> mapWriteAsync();
-    void unmap();
+{{GPUBuffer}} has the following internal slots:
 
-    void destroy();
-};
-</script>
+<dl dfn-type=attribute dfn-for="GPUBuffer">
+	: <dfn>\[[size]]</dfn>
+	::
+        The Length of the {{GPUBuffer}} allocation in bytes, which is an unsigned long long.
+
+	: <dfn>\[[usage]]</dfn>
+	::
+        The allowed usages for this {{GPUBuffer}}, represented as a bitset unsigned long.
+
+    : <dfn>\[[state]]</dfn>
+    ::
+        This {{GPUBuffer}}'s state which is one of `"unmapped"`, `"mapped"`, `"destroyed"`.
+</dl>
+
+## Creation ## {#buffer-creation}
 
-### Creation ### {#buffer-creation}
+A {{GPUBuffer}} can be created unmapped using the {{GPUDevice/createBuffer(descriptor)}} method.
```

Comment:
Also when this renders it doesn't display the "GPUDevice" part. It would be nice to show that. `{{GPUDevice/createBuffer(descriptor)}} method of {{GPUDevice}}`?

Reply:
Yeah I tried to do that, but after 10 minuets I still wasn't able to get Bikeshed to do that :/

Reply:
You can try {{GPUDevice/createBuffer(descriptor)|GPUDevice.createBuffer(descriptor)}}.

Reply:
If going with dots, then another alternative would be `{{GPUDevice}}.{{GPUDevice/createBuffer(descriptor)}}`.